cupy/cuda/pinned_memory.pyx

```diff
@@ -211,17 +211,20 @@ cpdef PinnedMemoryPointer alloc_pinned_memory(Py_ssize_t size):
     return _current_allocator(size)
 
 
-cpdef set_pinned_memory_allocator(allocator=_malloc):
+cpdef set_pinned_memory_allocator(allocator=None):
     """Sets the current allocator.
 
     Args:
         allocator (function): CuPy pinned memory allocator. It must have the
             same interface as the :func:`cupy.cuda.alloc_alloc_pinned_memory`
             function, which takes the buffer size as an argument and returns
-            the device buffer of that size.
+            the device buffer of that size. When ``None`` is specified, the
+            default memory allocator is used (i.e., memory pool is disabled).
```

Comment:
I thought the default is now the one using memory pool, so the text would make users confused. How about to use wording like "no memory pool"?

Reply:
Thanks. As the set_allocator argument may be a memory pool or a managed allocator, I fixed with the term `raw memory allocator`.
https://github.com/kmaehashi/cupy/blob/dad9a4c05adac59e80fab6f902716af3dfb26e6f/cupy/cuda/memory.pyx#L399-L405